src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -431,10 +405,18 @@ QueryNetworkResult Plugin::QueryNetwork(const CNNNetwork& network,
     // Get ops after transformations and check if it's supported
     // Transformations might lead to the situation when single node is merged to multiple operations,
     // so we mark original op as supported only if all nodes that it was merged into are supported
+    bool wasNodeAlreadyChecked = false;
+    bool isSupported = false;
     for (auto&& op : ops) {
+        wasNodeAlreadyChecked = false;
+        isSupported = false;
```

Comment:
```
 bool isSupported = layerIsSupported(op);
 for (auto&& fusedLayerName : ngraph::getFusedNamesVector(op)) {
            if (InferenceEngine::details::contains(originalOpNames, fusedLayerName)) {
                if (isSupported) {
                    supported.emplace(fusedLayerName);
                } else {
                    unsupported.emplace(fusedLayerName);
                }
 }
```

Reply:
done

Reply:
I need reject your suggestion code changes because constant operation is processed incorrect. This operation was added in 'constants' container when fusedLayerName which related with this operation is not contained in originalOpNames.